cmd/run.go

```diff
@@ -18,7 +19,7 @@ var runCmd = &cobra.Command{
 and summarizing it. If no files are given, the current
 working directory is assumed.  Any directories will be
 walked recursively.`,
-	RunE: runCommandWithViperArgs,
+	Run: runCommandWithViperArgs,
```

Comment:
What does the change from `runner returning error` to `runner exiting with 1` bring us ?
With the change the runner now has to print the error by itself.

Reply:
We don't want to print every error that the run command would return. It's rarely a good idea to use `RunE` with Cobra.

Reply:
Ok